app/models/mailing_list/steps/degree_stage.rb

```diff
@@ -15,7 +15,7 @@ def degree_status_options
       end
 
       def degree_status_option_ids
```

Comment:
This pattern is repeated a few times, maybe share it?

Reply:
I'll see if I can DRY it up in a follow up PR 👍